warp10/src/main/java/io/warp10/script/functions/LCSTORE.java

```diff
@@ -0,0 +1,80 @@
+//
```

Comment:
Merce LCSTORE with LSTORE by adding a flag to its constructor, the code being almost identical.